src/components/Artists/Artists.html

```diff
@@ -0,0 +1,23 @@
+<div class="artist">
```

Comment:
why class name is `artist` (singular) if component & file name is `artists` (plural)

Reply:
fixed